docs-source/usersguide/application.rst

```diff
@@ -1059,6 +1059,64 @@ sampling, and therefore is preferred for most applications.  Also note that
 :code:`LangevinIntegrator`\ , like :code:`LangevinMiddleIntegrator`\ , is a leapfrog
 integrator, so the velocities are offset by half a time step from the positions.
 
+Nosé-Hoover Integrator
+----------------------
+
+The :code:`NoseHooverIntegrator` uses the same "middle" leapfrog propagation
+algorithm as :code:`LangevinMiddleIntegrator`, but replaces the stochastic
+temperature control with a velocity scaling algorithm that produces more
+accurate transport properties :cite:`Basconi2013`.  This velocity scaling
+results from propagating a chain of extra variables, which slightly reduces the
+computational efficiency with respect to :code:`LangevinMiddleIntegrator`.  The
+thermostated integrator is minimally created with syntax analogous to the
+:code:`LangevinMiddleIntegrator` example above::
+
+    NoseHooverIntegrator integrator(300*kelvin, 25/picosecond,
```

Comment:
Is 25/ps a good choice?  How does this value relate to the friction coefficient?  For Langevin, we recommend a friction coefficient of 1/ps.

Reply:
Yes, that is a reasonable choice. Nosé-Hoover preserves the ensemble for position and momentum. In contrast, the Langevin thermostat introduces artificial damping through the (partial) randomization of velocities and therefore preserves the ensemble only for positions. As a consequence, the fricition coefficient must be small for Langevin in order to retain the kinetics. For Nose-Hoover, the collision frequency can be chosen according to (mainly) numerical considerations such as speed of convergence and stability.

Reply:
I would say it differently.  Langevin has an implicit heat bath (represented by the random and damping forces) while Nose-Hoover models the heat bath explicitly.  But either way the heat bath creates friction on the physical degrees of freedom, and the stronger that friction, the more it will slow them down.

I ran some tests to measure the effect of this.  I simulated alanine dipeptide in explicit solvent for 1 ns, and computed the autocorrelation functions for the phi and psi angles.  I repeated this with friction coefficients of 1/ps and 25/ps.  For phi, the higher friction only slowed it down a little:

![image](https://user-images.githubusercontent.com/4379786/81617952-3fee0980-939b-11ea-87e3-217a88294fec.png)

But for psi it slowed things down quite a lot:

![image](https://user-images.githubusercontent.com/4379786/81618186-ac690880-939b-11ea-831e-c788507ba200.png)